backend/superface/grid/chat/channels/maps/discord.suma

```diff
@@ -0,0 +1,212 @@
+profile = "chat/channels@1.0"
```

Comment:
Are we also going to remove the `superface` folder from `serverless/integrations`?

Reply:
Yes when we remove serverless stuff.